Documentation/Contributors/BuildGuide/README.md

```diff
@@ -98,3 +99,46 @@ Here's the full set of scripts and what they do.
    * `test-webgl` - Runs only WebGL tests with [Karma](http://karma-runner.github.io/0.13/index.html).
    * `test-webgl-validation` - Runs all tests with [Karma](http://karma-runner.github.io/0.13/index.html) and enables low-level WebGL validation.
    * `test-release` - Runs all tests with [Karma](http://karma-runner.github.io/0.13/index.html) on the minified release version of built Cesium.
+   * `deploy-s3` - Deploys the built cesium files, the npm package, and the zip file to Amazon S3. This requires having credentials set up for the S3 bucket to which you are deploying.
+   * `deploy-status` - Set the deployment statuses in GitHub, for use with Travis.
+   * `deploy-set-version` - Sets the version of `package.json`, for use with Travis.
+
+## Travis and Continuous Integration
+
+Cesium uses [Travis](https://travis-ci.org/) for continuous integration. The Travis configuration and all the steps of the build process are defined in `travis.yml`. The blog post [Cesium Continuous Integration](http://cesiumjs.org/2016/04/07/Cesium-Continuous-Integration/) contains an in-depth explaination of the travis build process.
+
+After pushing code to the Cesium repository or when opening a pull request, the build is triggered. After the build has completed, at the bottom on the pull request, the status of the build is shown and you can access the build by clicking the "Details" link.
+
+![Checks](checks_failed.jpg)
+
+You can also access the build of any branch of cesium by going to the [Cesium Branches](https://github.com/AnalyticalGraphicsInc/cesium/branches/all) page, and clicking the icon next to the branch name.
+
+![Branches](branches.png)
+
+Additional set up is required for deployment if you do not have commit access to Cesium.
+
+### Configure S3 Credentials
+
+To configure Travis for deployment for a fork of Cesium, you must have valid credentials to the cesium-dev bucket.
```

Comment:
They could also edit `travis.yml` to write a different bucket entirely, if they do this they would also have to set the `travisDeployUrl` variable at the top of gulpfile.js.  It's must more likely someone will set up their fork this way since anyone with credentials to `cesium-dev` most likely works in the main branch.